projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -5437,5 +5437,308 @@ public string PayorTypeFinancialClassHelper
                 }
             }
         }
+
+        /// <summary>Set an emerging issue value, creating an empty EmergingIssues Observation as needed.</summary>
+        private void SetEmergingIssue(string identifier, string value)
+        {
+            if (String.IsNullOrEmpty(value) && EmergingIssues == null)
+            {
+                return;
+            }
+            if (EmergingIssues == null)
+            {
+                EmergingIssues = new Observation();
```

Comment:
instead of defining the observation, use the new CreateObservation helper function